src/main/java/org/gridsuite/study/server/networkmodificationtree/NetworkModificationNodeInfoRepositoryProxy.java

```diff
@@ -77,11 +78,11 @@ public void handleExcludeModification(AbstractNode node, UUID modificationUuid,
     }
 
     @Override
-    public void removeModificationToExclude(AbstractNode node, UUID modificationUuid) {
+    public void removeModificationsToExclude(AbstractNode node, List<UUID> modificationUuid) {
         NetworkModificationNode networkModificationNode = (NetworkModificationNode) node;
         if (networkModificationNode.getModificationsToExclude() != null &&
-            networkModificationNode.getModificationsToExclude().contains(modificationUuid)) {
-            networkModificationNode.getModificationsToExclude().remove(modificationUuid);
+            networkModificationNode.getModificationsToExclude().containsAll(modificationUuid)) {
```

Comment:
modificationsToExclude list will not necessarily contain all modificationUuid of the `modificationUuid` list. This test is not correct. Maybe check **for each modification to remove** if it is present in the modificationsToExclude list. Or better, determine the intersection of the modificationsToExclude and modificationUuid lists and remove elements of this list.

Reply:
if the test was only to check presence before removing it, it is useless, we search 2 times removed test